### PR TITLE
Flatten the SNMP trap to the record level

### PR DESCRIPTION
**Pull request by jovillas**
Instead of parsing the trap and splitting some fields manually, just let
all of them be forwarded to the next plugin.

### CODE REVIEW

**Comment by Gabedlm** [comment]
Has this been tested on INT with the emulators ? and/or on Canary with real traps?

File: lib/fluent/plugin/filter_process_snmptrap.rb [hunk 1]
```diff
@@ -59,68 +19,22 @@ def start
...
-      determineEvent(record)
-      if record["event"].nil?
-        record["error"] << " : Can not determine the event"
-      end
-    end
+      # Replace invalid characters
+      fixedRecord = Hash.new
+      record.each { |recKey, recValue|
+          newKey = recKey
+          invalidChars.each { |invalidKey, subChar|
```
Comment (PakDLiu) [inline lib/fluent/plugin/filter_process_snmptrap.rb]: why is invalidChars in the test file?

Comment (jovillas) [reply lib/fluent/plugin/filter_process_snmptrap.rb]: Because I wanted it to be defined on the config.

**Comment by jovillas** [comment]
Yes it was tested.

**Pull request merged**